charts/spiffe-enable/templates/_helpers.tpl

```diff
@@ -0,0 +1,73 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "spiffe-webhook.name" -}}
```

Comment:
Looks like this was generated using the name spiffe-webhook. These should be updated to spiffe-enable.

Reply:
Updated throughout